### ISSUE TITLE
intro.js type thing for when a user first opens the TUI

### ISSUE THREAD

**Issue by njbrake** [njbrake/agent-of-empires#1564]
When someone first installs aoe and runs the tui, lets have a little bit more sophisticated walkthrough pages where we tell them how to use aoe, let them pick a theme, etc. It should be both keyboard navigatable and also clickable.
